packages/flutter_tools/test/general.shard/forbidden_imports_test.dart

```diff
@@ -67,6 +67,7 @@ void main() {
       fs.path.join(flutterTools, 'lib', 'src', 'test', 'flutter_platform.dart'),
```

Comment:
Can you remove some things from the whitelist?

Reply:
Removed runner.dart from it.